docs_2.0/manual_EN/3.quick-start/4.nebua-graph-crud.md

```diff
@@ -0,0 +1,251 @@
+    Execution succeeded (time spent 2817/3280 us)
+    ```
+
+2. Check the partition distribution with `SHOW HOSTS` to make sure the partitions are distributed in a balanced way.
+
+    ```nGQL
+    (root@nebula) [(none)]> SHOW HOSTS;
+    +-----------+-------+--------+--------------+---------------------+------------------------+
+    | Host      | Port  | Status | Leader count | Leader distribution | Partition distribution |
+    +-----------+-------+--------+--------------+---------------------+------------------------+
+    | storaged0 | 44500 | ONLINE | 1            | nba:5               | nba:5                  |
+    +-----------+-------+--------+--------------+---------------------+------------------------+
+    | storaged1 | 44500 | ONLINE | 1            | nba:5               | nba:5                  |
+    +-----------+-------+--------+--------------+---------------------+------------------------+
+    | storaged2 | 44500 | ONLINE | 1            | nba:5               | nba:5                  |
+    +-----------+-------+--------+--------------+---------------------+------------------------+
+    Got 3 rows (time spent 744/1123 us)
+    ```
+
+    If the **Leader distribution** is uneven, use `BALANCE LEADER` to redistribute the partitions. For more information, see [BALANCE](TODO) [TODO].
+
+3. Use the `nba` graph space.
+
+    ```nGQL
+    (root@nebula) [(none)]> USE nba;
+    Execution succeeded (time spent 1322/2206 us)
+    ```
+
+    You can use `SHOW SPACES` to check the graph space you created.
+
+    ```nGQL
+    (root@nebula) [nba]> SHOW SPACES;
+    +------+
+    | Name |
+    +------+
+    | nba  |
+    +------+
+    Got 1 rows (time spent 1235/1934 us)
+    ```
+
+## Create tags and edge types
+
+### nGQL Syntax
+
+```nGQL
+CREATE {TAG | EDGE} {<tag name> | <edge type>}(<property name 1> <data type>, [<property name 2> <data type>, ...]);
+```
+
+### Demonstration
+
+Create tags `player` and `team`, edge types `follow` and `serve`.
+
+| Component name | Type      | Property                         |
+| -------------- | --------- | -------------------------------- |  |
+| player         | Tag       | name (string), age (int)         |
+| team           | Tag       | name (string)                    |
+| follow         | Edge type | degree (int)                     |
+| serve          | Edge type | start_year (int), end_year (int) |
+
+```nGQL
+(root@nebula) [nba]> CREATE TAG player(name string, age int);
+Execution succeeded (time spent 2694/3116 us)
+
+Thu, 15 Oct 2020 06:22:29 UTC
+
+(root@nebula) [nba]> CREATE TAG team(name string);
+Execution succeeded (time spent 2630/3002 us)
+
+Thu, 15 Oct 2020 06:22:37 UTC
+
+(root@nebula) [nba]> CREATE EDGE follow(degree int);
+Execution succeeded (time spent 3087/3467 us)
+
+Thu, 15 Oct 2020 06:22:43 UTC
+
+(root@nebula) [nba]> CREATE EDGE serve(start_year int, end_year int);
+Execution succeeded (time spent 2645/3123 us)
+
+Thu, 15 Oct 2020 06:22:50 UTC
+```
+
+## Create indexes
```

Comment:
**DO NOT** put index ops at place!
otherwise almost every user will create an index at this place.
It is redundant and bad performance.
follow the order of 1.0 manual.

Reply:
Thank you for the heads-up. Using indexes can be really tricky. I put this section to the base and added a must-read subsection.